.github/workflows/publish.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Build and push Docker image to GHCR
         env:
           IMG_REPO: ${{ env.IMAGE_NAME }}
-          IMG_TAG: ${{ join(fromJSON(steps.meta.outputs.json).tags, ';') }}
+          IMG_TAG: ${{ join(fromJSON(steps.meta.outputs.json).tags, ' ') }}
           PUSH_REPO: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
         run: |
-          make docker-build
+          yes | make docker-build
```

Comment:
@sivanov-nuodb Are you sure that this works?
`yes` prints `y`, but the `docker/build.sh` requires the user to enter `yes`.